NimbleTests/AsynchronousTest.swift

```diff
@@ -39,6 +39,9 @@ class AsyncTest: XCTestCase {
             done()
         }
         waitUntil { done in
+            done(nil, 0, "", NSNumber.self, {})
```

Comment:
Nitpick: I think this would be better in a separate test case that explains its purpose.